specs/1125-KEYD-key_details.md

```diff
@@ -0,0 +1,9 @@
+## Key details
+
+- The lozenges shown link to that market in Console (<a name="1125-KEYD-001" href="#1125-KEYD-001">1125-KEYD-001</a>)
+- There is a warning to remember that if I hold an open position the balance / totals may not be accurate as is constantly changing (<a name="1125-KEYD-002" href="#1125-KEYD-002">1125-KEYD-002</a>)
+- I can see the balance of each (the sum across ALL account types) (<a name="1125-KEYD-003" href="#1125-KEYD-003">1125-KEYD-003</a>)
+- There is a button / icon that allows me to expand the view to show the breakdown of all non-zero accounts for that asset (<a name="1125-KEYD-004" href="#1125-KEYD-004">1125-KEYD-004</a>)
+- There is a way to switch between keys (or to easily navigate back to the keys page to achieve this) (<a name="1125-KEYD-005" href="#1125-KEYD-005">1125-KEYD-005</a>)
+<!-- Cannot test this as I cannot trigger the menu to open -->
```

Comment:
is this not significant? This is partly a question but also a note to self to have a look during testing

Reply:
Figured out how to test this now